tests/test_integration.py

```diff
@@ -18,7 +18,14 @@
 from deeprankcore.utils.exporters import HDF5OutputExporter
 from deeprankcore.utils.grid import GridSettings, MapMethod
 
-from . import PATH_TEST
+pdb_path = str("tests/data/pdb/3C8P/3C8P.pdb")
```

Comment:
Same as other comments here, I think is better to use PATH_TEST variable and os.join() method for defining paths